app/controllers/campaignController.js

```diff
@@ -0,0 +1,55 @@
+const postMessageMethod = 'chat.postMessage'
+const listUsersMethod = 'users.list'
+const Slack = require('slack-node')
+
+router.post('/test-send', function (req, res) {
+  url = getUrlForRequest(configResolver.getConfigVariable('TEST_USER'));
+  makeHttpsGetRequest(url);
+	res.send('"How was your week?" - notifications sent to private channels. Thanks!');
+})
+
+router.post('/test-send-many', function (req, res) {
+  setCampaign();
+  sendToMany();
+	res.send('"How was your week?" - notifications sent to private channels. Thanks!');
+})
+
+function setCampaign() {
+  campaignId = 'C' + Date.now();
+}
+
+function getUrlForRequest(username) {
+  return 'https://slack.com/api/' + postMessageMethod +
```

Comment:
good hint - thanks! I will also check if I can use `slack-node` lib. I turned away from it before as I couldn't make work attachments but then later I figured there needs to be different formatting so it might actually work. Will change this soon